elonacore.cpp

```diff
@@ -54415,7 +54332,7 @@ void proc_autopick()
                 }
             }
             elona::ci = ci;
-            do_open_command();
+            turn_result_t turn_result = do_open_command();
```

Comment:
Should the result of this command be taken into effect for the turn lifecycle? The player's turn can end early if the item being opened is a shopkeeper's trunk, deck or the fire giant's shackles, for example.